dotnet/samples/KernelSyntaxExamples/Example77_HandlebarsPromptSyntax.cs

```diff
@@ -0,0 +1,239 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.PromptTemplates.Handlebars;
+
+namespace KernelSyntaxExamples;
+
+// This example shows how to use Handlebars different syntax options like:
+//  - set (works)
+//  - function calling (works)
+//  - loops (each) - not working at the moment, but tries to do so over a complex object generated by one of the functions: a JSON array-
+//  - array - not working at the moment - to accumulate the results of the loop in an array
+//  - conditionals (works)
+//  - concatenation (works)
+
+// In order to create a Prompt Function that fully benefits from the Handlebars syntax power.
+// The example also shows how to use the HandlebarsPlanner to generate a plan (and persist it) which was used to generate the initial Handlebar template.
+// The example also shows how to create two prompt functions and a plugin to group them together.
+public static class Example77_HandlebarsPromptSyntax
+{
+    public static async Task RunAsync()
+    {
```

Comment:
Examples has been changed to integration tests today, look how it is done currently in repo.

Reply:
Thanks, @Krzysztof318! that's done - now it just needs a check/review in regards of the Handlebars template as discussed with @matthewbolanos in Discord - https://discord.com/channels/1063152441819942922/1194728332139442337/1196839845851582594  :)

Reply:
@joslat Hi, one more small fix yet :) Change Console to this.WriteLine. WriteLine from BaseTest is using ITestOutputHelper interface from xunit